app/ts/components/TransactionPage/TransactionDetails.tsx

```diff
@@ -102,8 +102,8 @@ type EthAmountProps = {
 }
 
 const EthAmount = ({ response, addFavoriteStore }: EthAmountProps) => {
-	if (response.value.eq(0)) return <></>
-	const ethAmount = ethers.utils.formatEther(response.value)
+	if (response.value === 0n) return <></>
```

Comment:
why we don't want to show zero balance here?

Reply:
I think leaving it out simplifies the UI when there wasn't attached value.

Reply:
intentionally left it out as there would most likely be a Token Amount if there is zero ETH amount. would simplify the UI to only show what's relevant for the transaction